README.md

```diff
@@ -4,6 +4,19 @@ A simple web application for a OpenAI-enabled document search. This repo uses Az
 
 ![Architecture](docs/architecture.png)
 
+## Learning More about Enterprise QnA 
+
+Enterprise QnA is built on a pattern the AI community calls "Retrieval-Augmented Generation" (RAG). In addition to this repository having a reference architecture on how to implement this pattern on Azure, here are resources to familiarize yourself with the concepts in RAG, and samples to learn each underlying product's APIs:
+
+| Resource | Links	| Purpose| Highlights | 
+| ---- | ----	| ---- | ----------------- | 
+| Reference Architecture |	[GitHub](https://github.com/ruoccofabrizio/azure-open-ai-embeddings-qna) (This Repo) | 	Starter template for enterprise development.	- Easily deployable reference architecture following best practices. |  -  Frontend is Azure OpenAI chat orchestrated with Langchain. <br> - Composes Form Recognizer, Azure Search, Redis in an end-to-end design. <br>	- Supports working with Azure Search, Redis. |
+|  Educational Blog Post|	[Microsoft Blog](https://github.com/ruoccofabrizio/azure-open-ai-embeddings-qna), <br> [GitHub](https://github.com/Azure-Samples/azure-search-openai-demo/) |	Learn about the building blocks in a RAG solution.	| - Introduction to the key elements in a RAG architecture. <br> - Understand the role of vector search in RAG scenarios.<br> - See how Azure Search supports this pattern. <br> - Understand the role of prompts and orchestrator like Langchain. |
```

Comment:
link should be https://techcommunity.microsoft.com/t5/ai-applied-ai-blog/revolutionize-your-enterprise-data-with-chatgpt-next-gen-apps-w/ba-p/3762087